setup.py

```diff
@@ -301,6 +300,7 @@ def merge_deploy_libfuzzer_sanitizer(self, libfuzzer, lib_name,
     description="A coverage-guided fuzzer for Python and Python extensions.",
     long_description=open("README.md", "r").read(),
     long_description_content_type="text/markdown",
+    packages=["atheris", "atheris_no_libfuzzer"],
```

Comment:
I'm wondering if we should maybe make the two C extensions submodules of `atheris`. Of course, you'd have to spell the no-libfuzzer versions something like `atheris.no_libfuzzer.Fuzz()`. But, if we're going to make a breaking change, now's the time.